prefix.go

```diff
@@ -65,7 +63,7 @@ func (i *Ipamer) DeletePrefix(cidr string) (*Prefix, error) {
 func (i *Ipamer) AcquireChildPrefix(parentCidr string, length int) (*Prefix, error) {
 	var prefix *Prefix
 	var err error
-	retryOnOptimisticLock(func() error {
+	_ = retryOnOptimisticLock(func() error {
```

Comment:
the `retryOnOptimisticLock` returns an error, so why don't you simply omit the declaration of the `err` and write
~~~
return prefix, retryOnOptimisticLock(....)
~~~
in this case you also remove the ugly underscore assignment ...